src/crimson/os/seastore/extent_placement_manager.cc

```diff
@@ -987,7 +987,17 @@ RandomBlockOolWriter::alloc_write_ool_extents(
     return alloc_write_iertr::now();
   }
   return seastar::with_gate(write_guard, [this, &t, &extents] {
-    return do_write(t, extents);
+    std::shared_ptr<rbm_pending_ool_t> ptr = std::make_shared<rbm_pending_ool_t>();
+    ptr->pending_extents = t.get_pre_alloc_list();
```

Comment:
Done.

Reply:
`ptr->pending_extents` can differ from `extents` because the `extents` is generated from `get_valid_pre_alloc_list`, as shown below.

```
    auto get_valid_pre_alloc_list() {
      std::list<CachedExtentRef> ret;
      assert(num_allocated_invalid_extents == 0);
      for (auto& extent : pre_alloc_list) {
        if (extent->is_valid()) {
          ret.push_back(extent);
        } else {
          ++num_allocated_invalid_extents;
        }
      }
      for (auto& extent : pre_inplace_rewrite_list) {
        if (extent->is_valid()) {
          ret.push_back(extent);
        }
      }
      return ret;
    }
```